strapi/strapi
# [Blocks] Replace alpha warnings by beta warnings

### What does it do?

Replaces Blocks alpha warnings by blocks beta warnings.

The color of the alert in the CTB modal forms doesn't match the one in the design. But in the design they use a variant using the secondary (light blue) color that doesn't exist in the design system. I checked with @lucasboilly, it's okay to use the default (primary blue) Alert variant instead.
### Why is it needed?

We're releasing the beta soon
### How to test it?

Add a blocks field, edit content in a blocks field. You should see blue beta warnings instead of orange alpha warnings.
### Related issue(s)/PR(s)

#18151 initially added the alpha warnings
